testDB/users.db

```diff
@@ -1 +1,2 @@
+{"username":"eugenio","password":"erikiado123","apiToken":"082be9526329ebea51d69e11a50217d93f21545d","_id":"V6DEwzQASu52PWI8"}
```

Comment:
I think you need to delete the TestDB when you finish your tests. 

And Don't forget to run lint.